next.clayui.com/src/components/clay/Tooltip.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import React from 'react';
+
+import Editor from './Editor';
+
+// Workaround for building gatsby's SSR with Tooltip
+const {ClayTooltipProvider, default: ClayTooltip} =
```

Comment:
Yeah, that sounds weird but if there is no other solution 🤷‍♂️. Maybe we can just ignore the `metal-position` in `onCreateWebpackConfig.js`, could that solve?

```js
{
	loader: 'null-loader',
	test: /clay-tooltip|metal-clipboard|metal-position/,
},
```

Reply:
I tried that and it didn't work, it just caused another error where a component was calling `Align`